d1lod/d1lod/sesame/interface.py

```diff
@@ -185,13 +175,20 @@ def insertModel(self):
             print "Attempted to insert a model that was None."
             return
 
+        # checking for all the statements in the current model
+        # if either subject / object / predicate is a blank node - then indicate that the payload contains a blank node
+        blank_node = False
+        for s in self.model:
+            blank_node = self.tripleHasBlankNode(s.subject, s.predicate, s.object)
+            if(blank_node == True):
+                break
+
         sparql_data = " .\n ".join([str(s) for s in self.model])
-        sparql_query = u"INSERT DATA { %s }" % sparql_data
 
         # Log model size
         logging.info('Inserting model of size %d.', self.model.size())
 
-        self.repository.update(sparql_query)
+        self.repository.insert_data(repository_name=self.repository.name,payload=sparql_data, blank_node = blank_node)
```

Comment:
Extra space around arg

```
blank_node = blank_node
```

to

```
blank_node=blank_node
```

Reply:
Made conditions Python2 compliant.
Commit: https://github.com/ec-geolink/d1lod/pull/48/commits/be01976d391a3f32b6e6b63c6f6dfde7d57988cd